source/renderer/app/api/utils/patchAdaApi.js

```diff
@@ -73,11 +70,7 @@ export default (api: AdaApi) => {
         futureEpoch: {
           // N+2 epoch
           epochNumber: nextEpochNumber ? nextEpochNumber + 1 : null,
-          epochStart: nextEpochStartTime
-            ? moment(nextEpochStartTime)
```

Comment:
@yakovkaravelov why did you remove this? This is start time of the N+2 epoch and not the N+1 (next) one?

Reply:
@nikolaglumac I didn't remove this.
This is calculated in another file.

Reply:
Not as far as I can see: https://github.com/input-output-hk/daedalus/pull/2165/files#diff-6aa9e91ae277a5d21b8e1bb0b03ffafbR1633

Reply:
@nikolaglumac Same as above comment.

Reply:
@yakovkaravelov https://github.com/input-output-hk/daedalus/pull/2165/files#r484325653